### ISSUE TITLE
no batch job logs on Terrascope due to missing log.offset

### ISSUE THREAD

**Issue by bossie** [Open-EO/openeo-geopyspark-driver#947]
Recent log documents lack the `log.offset` property which our query relies on.

Was fixed on CDSE by [adding a random one at write time](https://teams.microsoft.com/l/message/19:c462a30b534647ef86cf90b6581f6820@thread.tacv2/1701091215191?tenantId=9e2777ed-8237-4ab9-9278-2c144d6f6da3&groupId=befdddf8-55e8-4c70-855e-8802739f2ae9&parentMessageId=1701091215191&teamName=Remote%20Sensing%20Services&channelName=openeo-devs&createdTime=1701091215191)

This is not k8s so maybe it can simply be enabled: https://vector.dev/docs/reference/configuration/sources/file/#offset_key: TBC

**Comment by JeroenVerstraelen** [comment]
@bossie is this still an issue?

**Comment by bossie** [comment]
Yes, because the short-term fix was to simply roll back the changes that broke the logs (= move from filebeat to vector on Terrascope).

The actual fix is to get `log.offset` in the ES log documents but that's a devops issue.

**Comment by bossie** [comment]
@JeroenVerstraelen: @tcassaert and @pvbouwel are better suited to take on this issue.

**Comment by tcassaert** [comment]
@bossie the `log.offset` was added.

**Comment by bossie** [comment]
Ok so that leaves restoring your vector-related changes that Niels B. temporarily reverted.

**Comment by tcassaert** [comment]
That already happened, everything should be coming into ES via Vector now.

**Comment by bossie** [comment]
Oh, cool. My test job on openeo-dev seems to return logs so 👍

**Comment by tcassaert** [comment]
Then we can close this issue I suppose, thanks for verifying!
